code/modules/mob/living/carbon/human/species.dm

```diff
@@ -2185,3 +2191,367 @@ GLOBAL_LIST_EMPTY(features_by_species)
  */
 /datum/species/proc/on_owner_login(mob/living/carbon/human/owner)
 	return
+
+/**
+ * Gets a short description for the specices. Should be relatively succinct.
+ * Used in the preference menu.
+ *
+ * Returns a string.
+ */
+/datum/species/proc/get_species_description()
+	SHOULD_CALL_PARENT(FALSE)
+	CRASH("Species [name] ([type]) did not have a description set, and is a selectable roundstart race! Override get_species_description.")
```

Comment:
```suggestion
	stack_trace("Species [name] ([type]) did not have a description set, and is a selectable roundstart race! Override get_species_description.")
	return "No species description, file a bug report!"
```

Just in case...